src/Middleware/Session/ref/Microsoft.AspNetCore.Session.csproj

```diff
@@ -10,5 +10,6 @@
     <Reference Include="Microsoft.Extensions.Caching.Abstractions"  />
     <Reference Include="Microsoft.Extensions.Logging.Abstractions"  />
     <Reference Include="Microsoft.Extensions.Options"  />
+    <Reference Include="Microsoft.Net.Http.Headers"  />
```

Comment:
A direct reference shouldn't be needed here, you already added it to Http.Abstractions

Reply:
Changed